screen/PreviousResult/PrveiousResultScreen.js

```diff
@@ -1,13 +1,107 @@
-import React from 'react';
-import { View, Text, StyleSheet } from 'react-native';
+import React, { useState } from 'react';
+import {
+  View,
+  Text,
+  Button,
+  ActivityIndicator,
+  StyleSheet,
+} from 'react-native';
 
-import COLORS from '../../common/constants/COLORS';
+import StandardModal from '../../common/components/StandardModal';
+import TitleText from '../../common/components/TilteText';
+import UnitText from '../../common/components/UnitText';
+import ErrorMessage from '../../common/components/ErrorMessage';
 import FONT from '../../common/constants/FONT';
+import COLORS from '../../common/constants/COLORS';
+
+const SURVIVED = "YOU'RE SURVIVED";
+const FAILED = 'YOU WERE INFECTED';
+const KILOMETER = 'km';
+const TIME = 'm';
+
+const PreviousResultScreen = () => {
+  const [modalVisible, setModalVisible] = useState(true);
```

Comment:
```suggestion
  const [modalVisible, setModalVisible] = useState(false);
```
로 해야 밑에서 해당 데이터의 오류가 생겼을 때 모달이 정상 작동 될 거 같습니다.